docs/mocksupport.rst

```diff
@@ -797,3 +853,13 @@ FakedWBEMConnection
 
 .. autoclass:: pywbem_mock.FakedWBEMConnection
    :members:
+
+.. _`DMTFCIMSchema`:
+
+DMTFCIMSchema
```

Comment:
Because this section now contains both the overview from the module docstring and the actual class description, I suggest changing the heading to "DMTF CIM Schema download support" or similar.

Reply:
DONE